src/components/alert/Alert.tsx

```diff
@@ -1,27 +1,40 @@
 import React from 'react';
 import { DialogBase } from '../dialog';
-import PropTypes from 'prop-types';
 import { Heading3, TextBody } from '../typography';
 import { Box } from '../box';
 import { Button, BUTTON_LEVELS } from '../button';
 import omit from 'lodash.omit';
 import { Illustration60X60Error, Illustration60X60Info } from '@teamleader/ui-illustrations';
+import { DialogBaseProps } from '../dialog/DialogBase';
 
-const ALERT_TYPES = {
-  confirm: 'confirm',
-  destructive: 'destructive',
-  error: 'error',
-};
+interface AlertProps extends DialogBaseProps {
+  /** Object containing the props of the primary action (a Button, with level prop set to 'primary'). */
+  primaryAction: Action;
+  /** Object containing the the props of the secondary action (a Button). */
+  secondaryAction?: Action;
+  /** The title of the alert. */
+  title: string;
+  /** The body of the alert. */
+  body?: string;
+  /** The type of the alert. */
+  type?: ALERT_TYPES;
+}
+
+interface Action {
+  label: string;
+  onClick: () => void;
+}
```

Comment:
Q: Shouldn't this be `ButtonProps` (or a subset) ? 
I can imagine a use case where I would like one of the buttons to become `processing` for example